src/types.ts

```diff
@@ -295,13 +296,14 @@ export interface TransactionResponse {
     serviceFee?: number;
     feeCurrency: string;
     amlScreeningResult?: AmlScreeningResult;
+    operation: TransactionOperation;
     customerRefId?: string;
     amountInfo?: AmountInfo;
     feeInfo?: FeeInfo;
-    signedMessages?: SignedMessageResponse[];
```

Comment:
`signedMessages` is an optional field

Reply:
Same [comment](https://github.com/fireblocks/fireblocks-sdk-js/pull/96#discussion_r895103297).